static/app/routes.tsx

```diff
@@ -1276,214 +1276,126 @@ function buildRoutes() {
         component={make(
           () => import('sentry/views/organizationGroupDetails/groupEventDetails')
         )}
-        props={{
-          currentTab: Tab.DETAILS,
-          isEventRoute: false,
-        }}
       />
       <Route
-        path="replays/"
+        path={TabPaths[Tab.REPLAYS]}
         component={make(
           () => import('sentry/views/organizationGroupDetails/groupReplays')
         )}
-        props={{
-          currentTab: Tab.REPLAYS,
-          isEventRoute: false,
```

Comment:
Actually do we still need this `isEventRoute` thing for something?

Reply:
It's only used to generate `baseUrl` in `groupDetails.tsx` - https://github.com/getsentry/sentry/blob/master/static/app/views/organizationGroupDetails/groupDetails.tsx#L198-L207. I found a different way to do that without relying on this prop.

Reply:
yessssssssss